emojivoto-statefulset.yml

```diff
@@ -0,0 +1,173 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: emojivoto
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  creationTimestamp: null
+  name: emoji
+  namespace: emojivoto
+spec:
+  serviceName: emoji
+  selector:
+    matchLabels:
+      app: emoji-svc
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: emoji-svc
+    spec:
+      containers:
+      - env:
+        - name: GRPC_PORT
+          value: "8080"
+        image: buoyantio/emojivoto-emoji-svc:v6
+        name: emoji-svc
+        ports:
+        - containerPort: 8080
+          name: grpc
+        resources:
+          requests:
+            cpu: 100m
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: emoji-svc
+  namespace: emojivoto
+spec:
+  selector:
+    app: emoji-svc
+  clusterIP: None
+  ports:
+  - name: grpc
+    port: 8080
+    targetPort: 8080
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  creationTimestamp: null
+  name: voting
+  namespace: emojivoto
+spec:
+  serviceName: voting
+  selector:
+    matchLabels:
+      app: voting-svc
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: voting-svc
+    spec:
+      containers:
+      - env:
+        - name: GRPC_PORT
+          value: "8080"
+        image: buoyantio/emojivoto-voting-svc:v5
```

Comment:
Ahh, looks like the emojivoto-daemonset.yml config is using a mix of v5 and v6 images, but we should be using the v6 images throughout. Would you mind updating this file to use `buoyantio/emojivoto-voting-svc:v6` and making an equivalent change in emojivoto-daemonset.yml while you're at it?